### PR TITLE
Add light options of the 3D configuration dialog

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: docs/user_manual/introduction/qgis_gui.rst [hunk 1]
```diff
@@ -2415,10 +2415,14 @@ fine-tune the 3D scene:
...
-  particular position (in :guilabel:`X`, :guilabel:`Y` and
-  :guilabel:`Z`), :guilabel:`Color`, :guilabel:`Intensity` and
-  :guilabel:`Attenuation`.
+* :guilabel:`Lights`: you can add
+
+  * up to eight :guilabel:`Point lights`, each with a particular position
+    (at :guilabel:`X`, :guilabel:`Y` and :guilabel:`Z`), :guilabel:`Color`,
+    :guilabel:`Intensity` and :guilabel:`Attenuation`
+  * up to four :guilabel:`Directional lights`, with the line direction
+    (as :guilabel:`X`, :guilabel:`Y` and :guilabel:`Z`), :guilabel:`Color`,
```
Comment (DelazJ) [inline docs/user_manual/introduction/qgis_gui.rst:2424]: My math is quite old so I'd appreciate if someone could explain me how the x,y,z values define the line direction

Comment (havatv) [reply docs/user_manual/introduction/qgis_gui.rst:2424]: > My math is quite old so I'd appreciate if someone could explain me how the x,y,z values define the line direction

I assume that they define the light direction vector.
(1,0,0): along the first axis (x axis) - in the direction of that axis.
(0,0,-1): along the third axis (z axis) - in the oposit direction of the axis (from above?).

Comment (NEDJIMAbelgacem) [reply docs/user_manual/introduction/qgis_gui.rst:2424]: > > My math is quite old so I'd appreciate if someone could explain me how the x,y,z values define the line direction
> 
> I assume that they define the light direction vector.
> (1,0,0): along the first axis (x axis) - in the direction of that axis.
> (0,0,-1): along the third axis (z axis) - in the oposit direction of the axis (from above?).

Yes it is as you say each component is for a an axis in the 3D view

File: docs/user_manual/introduction/qgis_gui.rst [hunk 2]
```diff
@@ -2415,10 +2415,14 @@ fine-tune the 3D scene:
...
+    and :guilabel:`Intensity`
```
Comment (havatv) [inline docs/user_manual/introduction/qgis_gui.rst:2425]:
```suggestion
  * up to four :guilabel:`Directional lights`, with direction vector (:guilabel:`X`,
    :guilabel:`Y`, :guilabel:`Z`), :guilabel:`Color` and :guilabel:`Intensity`.
```

**Pull request merged**